src/plugins/advanced_settings/public/management_app/advanced_settings.test.tsx

```diff
@@ -268,6 +268,33 @@ describe('AdvancedSettings', () => {
     ).toHaveLength(1);
   });
 
+  it('should should not render a custom setting', async () => {
+    // For various complicated reasons the mock returns false for isConfig, override that
```

Comment:
@pgayvallet the whole `advanced_settings.test.ts` needs some serious rework. The test suite hasn't seen much "love" for a while and hacks around implementation within the `AdvancedSettings` component.
Reworking this test suite is tech debt and will take a while, which we can tackle in a larger effort.